sys/include/div.h

```diff
@@ -128,6 +186,65 @@ static inline uint64_t div_u64_by_15625div512(uint64_t val)
 }
 
 /**
+ * @brief Divide val by (625/576)
+ *
+ * This is used to quantize a 1MHz value to the closest 921600Hz value,
+ * e.g., for timers.
+ *
+ * @param[in]   val     dividend
+ * @return      (val / (625/576))
+ */
+static inline uint64_t div_u64_by_625div576(uint64_t val)
+{
+    if (val > 10485759996ull) {
+        /* this would overflow 2^64 in the multiplication that follows, need to
+         * use the long version */
+        return (_div_mulhi64(DIV_H_INV_625_64, val) * 576 >> (DIV_H_INV_625_SHIFT));
+    }
+
+    return (val * DIV_H_INV_625_32) * 576 >> (DIV_H_INV_625_SHIFT + 32);
+}
+
+/**
+ * @brief Divide val by (576/625)
+ *
+ * This is used to quantize a ticks value to the closest 921600 value,
+ * e.g., for timers.
+ *
+ * @param[in]   val     dividend
+ * @return      (val / (576/625))
+ */
+static inline uint32_t div_u64_by_576div625(uint64_t val)
```

Comment:
look inconsistent, why is it returning a `uint32_t` while all the other (similar) functions `div_u64_by_xyz(...)` return `uint64_t`.